linode_api4/groups/lke.py

```diff
@@ -131,7 +131,7 @@ def cluster_create(
 
         result = self.client.post(
             "/lke/clusters",
-            data=_flatten_request_body_recursive(drop_null_keys(params)),
+            data=drop_null_keys(_flatten_request_body_recursive(params)),
```

Comment:
`drop_null_keys(...)` needed to be moved outside of the `_flatten_request_body_recursive(...)` call because it doesn't have handling for JSONObjects.